types/chroma-js/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "@types/chroma-js",
-    "version": "2.4.9999",
+    "version": "3.1.9999",
     "projects": [
```

Comment:
This should be marked as `"type": "module"` to match upstream.

I am not entirely certain why ATTW doesn't complain about this particular thing, honestly.

Reply:
Notably, this will require a flip in extensions, which ends up matching what upstream says, which is good.